core/client/pom.xml

```diff
@@ -37,6 +37,10 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
we should always include direct imports in the module pom, rather than relying on the indirect dependencies such as alluxio-core-common